lib/Mojolicious/Plugin/OpenAPI.pm

```diff
@@ -253,6 +258,28 @@ sub _route_path {
   return $path;
 }
 
+sub _security_action {
+  my ($self, $security_settings) = @_;
+  my $security_cb = $self->_security_cb;
+
+  return sub {
+    my $c = shift;
+    my @cc
+      = map { my ($name, $config) = %$_; ($security_cb->{$name}, $config); } @$security_settings;
+    my $wrapper;
+
+    $wrapper = sub {
+      my $c = shift;
+      my $cb = shift @cc || sub { die 'No security handler defined' };
+      $c->$cb(shift @cc, $wrapper);
```

Comment:
@kivilahtio: Both `$c->openapi->security->dummy` and `$c->openapi_security_dummy` are helpers.